File: server/src/main/java/com/grpc/empuje_comunitario/domain/messages/TransferDonationModel.kt
package com.grpc.empuje_comunitario.domain.messages

data class TransferDonationModel(
    // ID de solicitud (a la que se responde)
    val requestId: String,
    // ID de la organización donante
    val donationOrgId: String,
    val donations: List<DonationItemModel>
)

File: client/EmpujeComunitario.Client/EmpujeComunitario.Client.Common/Model/MessagesRabbitMq/TransferDonationModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace EmpujeComunitario.Client.Common.Model.MessagesRabbitMq
{
    public class TransferDonationModel
    {
        public Guid RequestId { get; set; }
        public Guid DonationOrgId {  get; set; }
        public List<DonationItemModel> Donations = new List<DonationItemModel>();
    }
}
